rust-toolchain

```diff
@@ -1 +0,0 @@
-1.51.0
```

Comment:
Why this has been deleted ? It is useful when an user clones and compiles this repo manually

Reply:
Bringing this on back